src/org/zaproxy/zap/extension/api/GoAPIGenerator.java

```diff
@@ -69,16 +70,27 @@ protected void generateAPIFiles(ApiImplementor imp) throws IOException {
 		String className = imp.getPrefix().substring(0, 1).toUpperCase() + imp.getPrefix().substring(1);
 		String pkgName = safeName(camelCaseToLowerCaseDash(className));
 
-		Path file = getDirectory().resolve(pkgName + ".go");
-		System.out.println("Generating " + file.toAbsolutePath());
-		try (BufferedWriter out = Files.newBufferedWriter(file, StandardCharsets.UTF_8)) {
+		Path path = getDirectory().resolve(pkgName + "_generated.go");
```

Comment:
Probably better to leave the variable name as `file`, it describes better what the `Path` is representing.